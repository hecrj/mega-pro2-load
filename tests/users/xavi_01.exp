5  // nombre M de pelicules

5  // pes primera pelicula
15  
20  
25
30  // pes de l'ultima pelicula

10  // nombre N de servidors

9 3 2 6 0 5 0 0 0 4 0 0 1 8 10 0 0 0 7 0 0  //lectura en preordre amb marca 0

//  a continuacio venen les seves dades, comen√ßant pel servidor 1 fins al 10

10  // ample de banda del primer servidor
3 1 2 4 // les 3 pelicules del primer servidor

5
1 5

11
4 1 2 3 5

3
1 2

2
1 1

3
2 2 3

2
1 5

3
0

15
4 1 2 3 5

2
1 3  //dades ultim servidor


//Inici programa en si

-5 1 3 // Consulta la pelicula mes solicitada en l'interval [1..3]

-1 3 4 // Peticio de: Pelicula 4 a l'instant 4

-1 1 5 // Peticio de: Pelicula 1 a l'instant 5

-3 6 // Consulta els servidors ocupats a l'instant 6

-2 8 // Consulta les peticions actives a l'instant 8

-4 2 // Afegeix la peli 4 al servidor 2
1 4
0

-1 4 9 // Peticio de: Pelicula 4 a l'instant 9

-5 3 10 // Consulta quina es la pelicula mes solicitada a l'interval [3..10]

-6 // Fi del programa