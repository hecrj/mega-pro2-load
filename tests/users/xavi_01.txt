5  

5  
15  
20  
25
30  

10  

9 3 2 6 0 5 0 0 0 4 0 0 1 8 10 0 0 0 7 0 0  



10  
2 1 2

5
1 5

11
4 1 2 3 5

3
1 2

2
1 1

3
2 2 3

2
1 5

3
0

15
4 1 2 3 5

2
1 3  




-5 1 3 

-1 4 3 

-1 1 5 

-3 6 

-2 8 

-4 2 
1 4
0

-1 4 9 

-5 3 10 

-6 