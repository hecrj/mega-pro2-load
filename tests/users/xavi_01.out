Pelicula mas solicitada
0 0
Peticion procesada y servidores
0 0
Peticion procesada y servidores
1 1
9
Servidores ocupados
0
Peticiones pendientes
0
Peticion procesada y servidores
2 5
2
Pelicula mas solicitada
1 1
